pyaedt/generic/pdf.py

```diff
@@ -1,56 +1,208 @@
+import json
 import os
-import warnings
-
-from pyaedt import __version__
-from pyaedt import pyaedt_path
-from pyaedt.edb_core.general import convert_py_list_to_net_list
-from pyaedt.generic.clr_module import List
-from pyaedt.generic.clr_module import _clr
-from pyaedt.generic.clr_module import is_clr
-
-try:
-    _clr.AddReference("AnsysReport")
-    from AnsysReport import CreatePdfReport
-    from AnsysReport import ReportSpec
-except ImportError:
-    msg = "pythonnet or dotnetcore not installed. Pyaedt will work only in client mode."
-    warnings.warn(msg)
 
+from fpdf import FPDF
 
-class AnsysReport:
-    """Generate a pdf report."""
-
+from pyaedt import __version__
+from pyaedt.generic.constants import unit_converter
+
+
+class ReportSpec:
```

Comment:
If this feature is not supposed to be used in IronPython, I would suggest using dataclasses

Reply:
it's a good idea